### PR TITLE
Update prevent-end-user-interaction-microsoft-defender-antivirus.md

### PR DESCRIPTION
**Pull request by kurtsarens**
Pausing a scan is not supported on Win10

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/threat-protection/microsoft-defender-antivirus/prevent-end-user-interaction-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -64,6 +64,9 @@ See [Prevent users from locally modifying policy settings](configure-local-polic
 
 You can prevent users from pausing scans, which can be helpful to ensure scheduled or on-demand scans are not interrupted by users.
 
+>[!NOTE]
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-antivirus/prevent-end-user-interaction-microsoft-defender-antivirus.md:67]:
```suggestion
> [!NOTE]
```

File: windows/security/threat-protection/microsoft-defender-antivirus/prevent-end-user-interaction-microsoft-defender-antivirus.md [hunk 2]
```diff
@@ -64,6 +64,9 @@ See [Prevent users from locally modifying policy settings](configure-local-polic
...
+>This setting is not supported on Windows 10
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-antivirus/prevent-end-user-interaction-microsoft-defender-antivirus.md:68]:
```suggestion
> This setting is not supported on Windows 10.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by denisebmsft — APPROVED** [review]
Thank you @kurtsarens and @JohanFreelancer9

**Pull request merged**